src/components/Jobs/Jobs.js

```diff
@@ -1,86 +1,73 @@
-} from '@folio/stripes/components';
+import { AccordionSet } from '@folio/stripes/components';
 
-import Job from './components/Job';
-import EndOfList from './components/EndOfList';
-import jobsMocks from './jobsMocks'; // TODO: to be replaced with real data in further stories (UIDATIMP-23, UIDATIMP-27)
+import PreviewsJobsAccordion from './components/PreviewsJobsAccordion';
+import JobsListAccordion from './components/JobsListAccordion';
+import jobsMocks from './jobsMocks';
 
 import css from './Jobs.css';
 
-export default class Jobs extends React.Component {
+class Jobs extends React.Component {
   static propTypes = {
     stripes: stripesShape.isRequired,
-  };
+    intl: intlShape.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.connectedPreviewsJobsAccordion = props.stripes.connect(PreviewsJobsAccordion);
+  }
 
   checkDateIsToday = date => {
-    const { formatDate } = this.props.stripes.intl;
+    const { formatDate } = this.props.intl;
 
     return formatDate(new Date()) === formatDate(date);
   };
 
-  renderPreviewJobs() {
-    return (
-      <div>
-        {
-          jobsMocks.preview.map(job => (
-            <Job
-              key={job.jobExecutionHRID}
-              job={job}
-              checkDateIsToday={this.checkDateIsToday}
-            />
-          ))
-        }
-      </div>
-    );
-  }
-
-  renderRunningJobs() {
-    return (
-      <div>
-        {
-          jobsMocks.running.map(job => (
-            <Job
-              key={job.jobExecutionHRID}
-              job={job}
-              checkDateIsToday={this.checkDateIsToday}
-            />
-          ))
-        }
-      </div>
-    );
-  }
-
   render() {
-    const { formatMessage } = this.props.stripes.intl;
+    const { formatMessage } = this.props.intl;
+    const previewsLabelText = formatMessage({ id: 'ui-data-import.previewJobs' });
+    const runningLabelText = formatMessage({ id: 'ui-data-import.runningJobs' });
+    const noItemsMessage = formatMessage({ id: 'ui-data-import.emptyJobs' });
 
     return (
       <div className={css.jobsPane}>
         <AccordionSet>
-          <Accordion
-            label={formatMessage({ id: 'ui-data-import.previewJobs' })}
-            separator={false}
-          >
-            <div className={css.jobList}>
-              {this.renderPreviewJobs()}
-            </div>
-            <EndOfList />
-          </Accordion>
-          <Accordion
-            label={formatMessage({ id: 'ui-data-import.runningJobs' })}
-            separator={false}
-          >
-            <div className={css.jobList}>
-              {this.renderRunningJobs()}
-            </div>
-            <EndOfList />
-          </Accordion>
+          {/* <JobsListAccordion
+            jobs={[]}
+            checkDateIsToday={this.checkDateIsToday}
+            labelText={previewsLabelText}
+            hasLoaded
+            noItemsToShow={noItemsMessage}
+          /> */}
+          <PreviewsJobsAccordion
```

Comment:
I do not see a reason for introducing wrapper around `Accordion`. You are dealing with a list of data. Accordion component just stateless component for storing any content inside it. Your abstraction makes sense but do it without an accordion.